paf-mvp-frontend/src/main.ts

```diff
@@ -5,34 +5,44 @@ import { notificationService } from './services/notification.service';
 import { NotificationEnum } from '@frontend/enums/notification.enum';
 import { currentScript } from '@frontend/utils/current-script';
 import {
+  deleteIdsAndPreferences,
   generateSeed,
   getAuditLogByDivId,
   getAuditLogByTransaction,
   getIdsAndPreferences,
   getNewId,
-  refreshIdsAndPreferences,
   registerTransmissionResponse,
+  removeCookie,
+  setPromptHandler,
   signPreferences,
   updateIdsAndPreferences,
 } from './lib/paf-lib';
 import { Window } from './global';
 
 currentScript.setScript(document.currentScript as HTMLScriptElement);
 
-const promptConsent = () => new Promise<boolean>((resolve) => new PromptConsent({ emitConsent: resolve }).render());
-const showNotification = (type: NotificationEnum) => notificationService.showNotification(type);
-
-(<Window>window).PAFUI ??= { promptConsent, showNotification };
 (<Window>window).PAF = {
   ...((<Window>window).PAF ?? {}),
   // The rest has to be the official methods, should not be overridden from the outside
   getNewId,
   signPreferences,
+  deleteIdsAndPreferences,
+  removeCookie,
   getIdsAndPreferences,
-  refreshIdsAndPreferences,
   updateIdsAndPreferences,
+  setPromptHandler,
   generateSeed,
   registerTransmissionResponse,
   getAuditLogByTransaction,
   getAuditLogByDivId,
 };
+
+(async () => {
```

Comment:
> Should it be the UI that does such configuration?
Yes, it's the piece of code to declare itself to the PAF lib

> Until now, the main.ts was a more declarative file.
You are right. As matter of fact, I have updated it in a next commit, so that it is purely declarative.